docs/root/intro/arch_overview/listener_filters.rst

```diff
@@ -0,0 +1,19 @@
+.. _arch_overview_listener_filters:
+
+Listener filters
+================
+
+As discussed in the :ref:`listener <arch_overview_listeners>` section, listener filters may be
+used to manipulate connection metadata. In fact, Envoy implements :ref:`*use_original_dst* and
+*bind_to_port* listener options <config_listeners_v1>` internally by passing the newly accepted
+connection through listener filters implementing these functions. Main purpose of listener filters
+is to make adding further system integration functions easier by not requiring changes to Envoy
+core functionlity, and also makes interaction between multiple such features more explicit.
```

Comment:
typo "functionlity" (please run change through spell check)

Reply:
Will do.

Reply:
Done, did not find any more spelling mistakes.